tgui/packages/tgui/interfaces/BrigTimer.tsx

```diff
@@ -15,7 +15,7 @@ export const BrigTimer = (props) => {
   const { act, data } = useBackend<Data>();
   const { timing, minutes, seconds, flash_charging } = data;
   return (
-    <Window width={300} height={138}>
+    <Window width={350} height={138}>
```

Comment:
this seems a little too long now from the videos you showed, can you make this a little thinner?